cmd/infracost/run.go

```diff
@@ -326,7 +327,7 @@ func runProjectConfig(cmd *cobra.Command, runCtx *config.RunContext, ctx *config
 		NoColor:       runCtx.Config.NoColor,
 		Indent:        "  ",
 	}
-	spinner := ui.NewSpinner("Calculating monthly cost estimate", spinnerOpts)
+	spinner := ui.NewSpinner("Extracting only cost-related params from terraform plan\nRetrieving cloud prices to calculate costs", spinnerOpts)
```

Comment:
Should this be two separate spinners? The first one (`Extracting only cost-related params...`) inside the `LoadResources` call starting after the plan JSON has been generated, and the second one (`Retrieving cloud prices`) here.

Reply:
hmm in the implementation docs it's as such:

```
  ✔ Extracting only cost-related params from terraform plan
     Retrieving cloud prices to calculate costs
 ```
 
 paging dr @alikhajeh1 - thoughts?

Reply:
Ah okay, I left a comment on the original docs that suggested this option as two separate spinners, but it looks that comment was resolved so wasn't appearing any more.

I don't think having multi-lines works with our current spinner library anyway.

Reply:
👍  - I'll alter